recipe_scrapers/aldinord.py

```diff
@@ -0,0 +1,20 @@
+from ._abstract import AbstractScraper
+
+
+class AldiNord(AbstractScraper):
+    @classmethod
+    def host(cls, domain: str = "aldi-nord.de"):
+        return domain
+
+    def author(self):
+        return "ALDI"
+
+    def site_name(self):
+        return "ALDI"
```

Comment:
Suggestion 1 of 2: we can use `StaticValueException`s to indicate whether we retrieved a value from the source webpage, or whether it is a static/constant value we're returning from the code:

```suggestion
from ._abstract import AbstractScraper
from ._exceptions import StaticValueException

class AldiNord(AbstractScraper):
    @classmethod
    def host(cls, domain: str = "aldi-nord.de"):
        return domain

    def author(self):
        raise StaticValueException(return_value="ALDI")

    def site_name(self):
        raise StaticValueException(return_value="ALDI")
```